Sources/RestBird-Alamofire/AlamofireSessionManager.swift

```diff
@@ -87,25 +111,33 @@ extension AlamofireSessionManager {
                                                   to: urlWithBaseUrl(baseUrl, request: request),
                                                   method: request.method.alamofireMethod,
                                                   headers: request.headers?.mapValues { String(describing: $0) })
-        }
-
-        if let request = uploadRequest.request {
-            do {
-                try delegate?.sessionManager(self, willPerform: request)
-            } catch {
-                completion(.failure(error))
+        case .multipart(let name, let fileName, let mimeType):
```

Comment:
There may be a better approach for this. We either specify them at project level and not on RestBird, we can create a MimeType object on library level and use it from there, or we can determine it from the data object (https://stackoverflow.com/questions/21789770/determine-mime-type-from-nsdata). Let me know which seems the best approach to you.

Reply:
A default mime type can be determined based on the data, but we should add a possibility to change it